prometheus/src/main/scala/filodb/prometheus/ast/Functions.scala

```diff
@@ -12,64 +13,100 @@ trait Functions extends Base with Operators with Vectors {
       RangeFunctionId.withNameLowercaseOnlyOption(name.toLowerCase).isEmpty &&
       FiloFunctionId.withNameLowercaseOnlyOption(name.toLowerCase).isEmpty &&
       MiscellaneousFunctionId.withNameLowercaseOnlyOption(name.toLowerCase).isEmpty &&
+      ScalarFunctionId.withNameInsensitiveOption(name.toLowerCase).isEmpty &&
       SortFunctionId.withNameLowercaseOnlyOption(name.toLowerCase).isEmpty) {
+
       throw new IllegalArgumentException(s"Invalid function name [$name]")
     }
 
+    /**
+      *
+      * @return true when function is scalar or time
+      */
+    def isScalarFunction(): Boolean = {
+       name.equalsIgnoreCase("scalar") ||
```

Comment:
1) braces not really needed since it's just an expression
2) can we please make the body lines line up/indented at 2 spaces?